cmd/zetaclientd/debug.go

```diff
@@ -97,10 +97,10 @@ func DebugCmd() *cobra.Command {
 
 			if chains.IsEVMChain(chain.ChainId) {
 
-				ob := evm.ChainClient{
+				ob := evm.Client{
 					Mu: &sync.Mutex{},
 				}
-				ob.WithZetaBridge(bridge)
+				ob.WithZetacoreClient(client)
```

Comment:
Inconsistency: one is `WithZetacoreClient` other is `WithZetaCoreClient`

I think we could use `Zetacore` as a single word everywhere  (and `Zetaclient` as well)

Reply:
cool. Let's use the single word `zetacore` as the terminology